### DISCUSSION TITLE
Missing Dependencies /

### DISCUSSION THREAD

**Discussion by DraigNET** [OwlGamingCommunity/V#4] [Support]
Although most of the dependencies are located within the folders, the solution shows over 70 errors for missing assembly files, including rage-sharp.dll, libCodewalker.dll and others. Some I was able to fix by adding the references to the solution via Nuget, and copy/pasting the DLL to overwrite the one included in this source.

![rageerror](https://user-images.githubusercontent.com/66184380/128256721-4ac70d07-7b2e-4f56-bafa-f1186dac7be0.png)

**Comment by DraigNET** [comment]
I can compile owl_boot.dll but I get an error when trying to start the server
![owlerror](https://user-images.githubusercontent.com/66184380/128260519-92f570a5-ac58-42f9-a751-c50edbac1a9b.png)

**Comment by ChrisLetti** [comment]
Hello,

did you happen to fix the bug with the BaseScopeExtensions namespace? If so, please explain how you did it.

**Comment by Test83318** [comment]
Check that those dependency files isn't 1KB, because git by default doesn't download them. 
So be sure to use any system that supports LFS to get the files, as it could be the LFS extension for git or the github desktop client. I'm pretty sure it will solve all the problems with ease.
https://git-lfs.github.com/

**Reply by DraigNET** [reply]
Cloning with GitHub Desktop using Git LFS seemed to work. Now to get it running with the database lol

**Comment by DraigNET** [comment]
The documentation doesn't make a great lot of sense. I have the solution and all sub-directories in a folder called **'V'**.

Inside that I have a folder called **'Output\Debug\'**.

I've copy/pasted the contents of **server-files** to that Debug folder. Inside the Debug folder I have a folder called **'netcoreapp3.1'** which I assume is where the solution compiles the project too.

What do I need to put inside Config.Json in order for it to load the Owl Gaming gamemode instead of Freeroam?

**Reply by Test83318** [reply]
On the conf.json change:
    "csharp" : "disabled",
into:
    "csharp" : "enabled",

**Comment by DraigNET** [comment]
"maxplayers" : 100,
	"name" : "Mirage Roleplay", 
	"gamemode" : "mirage", 
	"stream-distance" : 500.0,
	"announce" : false,
	"csharp" : "enabled",
	"port": 22005


Does it matter what you put as the Gamemode?

**Reply by Test83318** [reply]
Yes, it must be one of the following list:
https://wiki.rage.mp/index.php?title=Gamemodes

**Comment by DraigNET** [comment]
Thanks. Everything seems to load but I'm just getting Access Denied messages for some reason.

**Reply by Test83318** [reply]
There are multiple ways to implement the database, the simplest is to change in the mysql project to hard code the database connection data.

(It must be done at both
Source\owl_mysql\ThreadedMySQL.cs => Lines starting at 640
Source\owl_mysql\Mysql.cs => Lines starting at 63
)

Remember that the mysql port by default is 3306

**Reply by DraigNET** [reply]
Yeah I did that. Changed the port, changed the settings in both files. Still get access denied.

**Reply by DraigNET** [reply]
Ran that Cooker.exe file and now it's working :)

**Reply by Gravy9588** [reply]
@Test83318 can you send me your discord
